python/lsst/ts/linearstage/csc.py

```diff
@@ -238,10 +248,9 @@ async def handle_summary_state(self):
                 else:
                     self.log.debug("Disabling motor")
                     await self.component.disable_motor()
-            except Exception as e:
+            except Exception:
                 err_msg = "Failed to enable or disable motor"
-                await self.fault(code=2, report=f"{err_msg}: {e}")
-                raise e
+                await self.fault(code=ErrorCode.DISABLE_MOTOR, report=f"{err_msg}")

```

Comment:
In all other cases you call `self.fault` with both the error message and the error except here. Please do that here too.

Reply:
No, I meant to always call self.fault everywhere with `report=f"{err_msg}: {e}` so please do that.